Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job stats:
job        count
-------  -------
vep_vcf        1
total          1

Select jobs to execute...
Execute 1 jobs...

[Wed May 22 11:03:51 2024]
localrule vep_vcf:
    input: /data2/mbaragli/python_app/app_python/work_dir/testrun_4_HG002/snv/testrun_4_HG002_longshot.vcf
    output: /data2/mbaragli/python_app/app_python/work_dir/testrun_log/snv/testrun_log_vep.vcf
    log: /data2/mbaragli/python_app/app_python/work_dir/testrun_log/logs/vep.log
    jobid: 0
    reason: Missing output files: /data2/mbaragli/python_app/app_python/work_dir/testrun_log/snv/testrun_log_vep.vcf
    wildcards: sample=testrun_log
    threads: 8
    resources: tmpdir=/tmp

vep -format vcf -i /data2/mbaragli/python_app/app_python/work_dir/testrun_4_HG002/snv/testrun_4_HG002_longshot.vcf -o /data2/mbaragli/python_app/app_python/work_dir/testrun_log/snv/testrun_log_vep.vcf --vcf --fork 8 --assembly GRCh38 --everything --cache --dir_cache /data2/mbaragli/tools/vep_plugins   --offline --fasta /data2/utility/references/genomics/Homo_sapiens_assembly38.fasta  >> /data2/mbaragli/python_app/app_python/work_dir/testrun_log/logs/vep.log
Activating conda environment: vep
Terminating processes on user request, this might take some time.
[Wed May 22 11:04:23 2024]
Error in rule vep_vcf:
    jobid: 0
    input: /data2/mbaragli/python_app/app_python/work_dir/testrun_4_HG002/snv/testrun_4_HG002_longshot.vcf
    output: /data2/mbaragli/python_app/app_python/work_dir/testrun_log/snv/testrun_log_vep.vcf
    log: /data2/mbaragli/python_app/app_python/work_dir/testrun_log/logs/vep.log (check log file(s) for error details)
    conda-env: vep
    shell:
        vep -format vcf -i /data2/mbaragli/python_app/app_python/work_dir/testrun_4_HG002/snv/testrun_4_HG002_longshot.vcf -o /data2/mbaragli/python_app/app_python/work_dir/testrun_log/snv/testrun_log_vep.vcf --vcf --fork 8 --assembly GRCh38 --everything --cache --dir_cache /data2/mbaragli/tools/vep_plugins   --offline --fasta /data2/utility/references/genomics/Homo_sapiens_assembly38.fasta  >> /data2/mbaragli/python_app/app_python/work_dir/testrun_log/logs/vep.log
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Complete log: .snakemake/log/2024-05-22T110351.270235.snakemake.log
WorkflowError:
At least one job did not complete successfully.
